Library/Homebrew/software_spec.rb

```diff
@@ -325,8 +326,8 @@ def select_download_strategy(specs)
 end
 
 class BottleSpecification
-  DEFAULT_PREFIX = "/usr/local".freeze
-  DEFAULT_CELLAR = "/usr/local/Cellar".freeze
+  DEFAULT_PREFIX = Homebrew::DEFAULT_PREFIX
+  DEFAULT_CELLAR = "#{DEFAULT_PREFIX}/Cellar".freeze
```

Comment:
Tempted to have a `HOMEBREW_DEFAULT_CELLAR`?

Reply:
I see only one use of `DEFAULT_CELLAR` in `software_spec.rb` and one place where it could be used in `system_config.rb`, so to me there doesn't seem to be a need.